content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -222,11 +222,11 @@ featurize:
 
 ### ⚙️ Expand to recall how it was generated.
 
-The `featurize` stage was created with this `dvc run` command. Notice the
+The `featurize` stage was created with this `dvc stage add` command. Notice the
 argument sent to the `-p` option (short for `--params`):
 
 ```cli
-$ dvc run -n featurize \
+$ dvc stage add -n featurize \
```

Comment:
Hmm. We probably needed to add `repro` somewhere here to keep the behavior the same.

Reply:
U: No I just checked https://dvc.org/doc/start/data-management/metrics-parameters-plots#expand-to-recall-how-it-was-generated and it's just a note so its OK. We could remove it and say you can just write the dvc.yaml contents (unrelated though).